docs/custom_datasources_index.rst

```diff
@@ -91,6 +91,14 @@ insights.specs.datasources.ssl_certificate
     :show-inheritance:
     :undoc-members:
 
+insights.specs.datasources.system_user_dirs
+--------------------------------------------------
```

Comment:
Would you please adjust the length of the `---` to the same as the above title?

Reply:
Of course, I overlooked it. Thanks!